.changeset/fuzzy-years-taste.md

```diff
@@ -0,0 +1,37 @@
+---
+'@clerk/backend': major
+'@clerk/nextjs': major
+'@clerk/remix': major
+'gatsby-plugin-clerk': major
+---
+
+## Breaking Changes
+
+### @clerk/backend
+
+Replace `Clerk` with `createClerkClient` [SDK-1058]
+
+
+```js
+import { Clerk } from '@clerk/backend';
+const clerk = Clerk({ secretKey: '...' });
+```
+
+now becomes:
+
+```js
+import { createClerkClient } from '@clerk/backend';
+const clerk = createClerkClient({ secretKey: '...' });
+```
+
+### @clerk/gatsby-plugin-clerk
```

Comment:
As per code it doesn't export `createClerkClient`, only `clerkClient`. So isn't this incorrect then?

Reply:
It exports it via `export * from '@clerk/backend';` later in the file.